.travis.yml

```diff
@@ -5,7 +5,7 @@
 sudo: false
 
 # Choose a lightweight base image; we provide our own build tools.
-language: c
+language: generic
```

Comment:
Why? I thought C was the default for ignoring this field on Travis. Never questioned it.